test/board_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship'
+require './lib/cell'
+require './lib/board'
+require 'pry'
+
+class BoardTest < Minitest::Test
+
+  def test_it_exists
+    board = Board.new
+
+    assert_instance_of Board, board
+  end
+
+  def test_it_has_attributes
+    board = Board.new
+
+    assert_equal 16, board.cells.length
```

Comment:
I'm not 100% sure this is testing an attribute and not the method cells. Can you think of a way to test attributes? I don't know if it has any that aren't explicitly tied to a method. Maybe rename test so it is more accurate...